Assets/Python/Victory.py

```diff
@@ -1848,12 +1848,16 @@ def onGreatPersonBorn(iPlayer, unit):
 				if pAztecs.getGreatGeneralsCreated() >= 3:
 					win(iAztecs, 1)
 					
-def onUnitPillage(iPlayer, iGold):
+def onUnitPillage(iPlayer, iGold, iUnit):
 
 	# third Viking goal: acquire 3000 gold by pillaging, conquering cities and sinking ships by 1500 AD
 	if iPlayer == iVikings:
 		if isPossible(iVikings, 2):
 			sd.changeVikingGold(iGold)
+			
+	elif iPlayer == iMoors:

```

Comment:
Oh, I see. The Moorish goal is intended to only count sunk ships, is the goal phrased differently?